src/index.js

```diff
@@ -27,6 +28,7 @@ async function bundleMDX(
     xdmOptions = (vfileCompatible, options) => options,
     esbuildOptions = options => options,
     globals = {},
+    cwd = undefined
```

Comment:
```suggestion
    cwd
```

This will default to `undefined` anyway right?